tool/python/singa/datasets/ae.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+from singa.model import * 
```

Comment:
the auto-encoder example uses the mnist dataset, hence no need to have a separate script for creating dataset?